docs/en/observability/application-logs.asciidoc

```diff
@@ -0,0 +1,219 @@
+[[application-logs]]
+= Application logs
+
+Application logs provide valuable insight into events that have occurred within your services and applications.
+Correlating these logs with application traces enables you to view the context in which a log event has occurred.
+
+Structured vs unstructured logs:
+
+Structured logging is where the application writes JSON objects directly--instead of
+parsing your logs into a structured object at ingest time.
+
+Why is structure so important in logs? So you can search and aggregate on individual fields within your logs.
+
+
+[float]
+[[ingest-application-logs]]
+== Ingest application logs
+
+There are four ways to ship your application logs to the {stack}.
+Your use case will determine the method that's right for you.
+
+[float]
+=== Plaintext logs with Filebeat
+
+**Requirements**
+
+* The Elastic APM agent for your programming language.
+* Raw, plain-text application logs
+* {filebeat}, to monitor the log files or locations that you specify, collects log events, filter and correlate multi-line logs, and forwards them to {es}
+
+**Pros**
+
+* All programming languages/frameworks are supported
+
+**Cons**
+
+* Must parse application logs to be useful—meaning writing and maintaining Grok patterns and spending CPU cycles on parsing
```

Comment:
```suggestion
* Must parse application logs to be useful—meaning writing and maintaining Grok patterns and spending CPU cycles on parsing
* Parsing is tied to the application log format, thus it can differ per application and needs to be maintained over time
* Log correlation requires to modify the application log format and inject IDs in log message.
```